### ISSUE TITLE
Ambiguous units in calculator 

### ISSUE THREAD

**Issue by ExploratoriumGPT** [genai-impact/ecologits#93]
### Description

I know that m is SI for meter, but it is easy to confuse for minutes, or even miles for those who sadly live in a non-metric nation. 
I think it would be clearer and more accessible to use the whole word 'meters'
![image](https://github.com/user-attachments/assets/8f58153e-aa6f-43c2-abc5-1686bf991351)

https://huggingface.co/spaces/genai-impact/ecologits-calculator
